transport/grpc/config_test.go

```diff
@@ -472,7 +472,7 @@ func TestTransportSpec(t *testing.T) {
 				if tt.wantInbound.ServerMaxRecvMsgSize > 0 {
 					assert.Equal(t, tt.wantInbound.ServerMaxRecvMsgSize, inbound.t.options.serverMaxRecvMsgSize)
 				} else {
-					assert.Equal(t, defaultServerMaxRecvMsgSize, inbound.t.options.serverMaxRecvMsgSize)
+					assert.Equal(t, 1024*1024*64, inbound.t.options.serverMaxRecvMsgSize)
```

Comment:
why do you need to change those 2? Test should pass without this change?

Reply:
It does pass without this change, though I was surprised that tests didn't fail when I changed the value. So added an explicit value to catch future intended changes.